storage/innobase/handler/ha_innodb.cc

```diff
@@ -17540,15 +17544,27 @@ ib_logf(
 	switch(level) {
 	case IB_LOG_LEVEL_INFO:
 		sql_print_information("InnoDB: %s", str);
+#ifdef HAVE_SYSTEMD
+		sd_notifyf(0, "STATUS=InnoDB: Info: %s", str);
+#endif
```

Comment:
1. STATUS messages show up in systemctl status output for the service. AFAIK they don't got to syslog however the call beside them doess.
2. ack - changed for fatal/error messages only.